src/popover.ts

```diff
@@ -373,8 +372,17 @@ export class Popover {
         const { addAuthorName } = Store.config;
 
         const type = (this.post.opReputation || 0) > 50 ? 'high' : 'low';
-        const comment = comments?.[type] ?? comments?.low;
-
+        let comment = comments?.[type] ?? comments?.low;
+        if (comment) {
+            const sitename = StackExchange.options.site.name || '';
+            const siteurl = window.location.hostname;
+            const questionId = StackExchange.question.getQuestionId().toString();
+
+            comment = comment.replace(/%SITENAME%/g, sitename);
+            comment = comment.replace(/%SITEURL%/g, siteurl);
+            comment = comment.replace(/%OP%/g, this.post.opName);
+            comment = comment.replace(/%QID%/g, questionId);
```

Comment:
Instead of reassigning multiple times, you can use consecutive calls to `.replace()` (properly aligned).